advanced/dapps/react-dapp-v2/src/constants/default.ts

```diff
@@ -272,6 +273,20 @@ export enum DEFAULT_KADENA_METHODS {
 }
 
 export enum DEFAULT_KADENA_EVENTS {}
+/**
+ * BITCOIN
+ */
+export enum DEFAULT_BIP122_METHODS {
+  // BIP122_SIGN_MESSAGE = "btc_signMessage",
+  // BIP122_SEND_TRANSACTION = "btc_sendTransaction",
+  BIP122_SEND_TRANSACTION = "sendTransfer",
+  BIP122_GET_ACCOUNT_ADDRESSES = "getAccountAddresses",
+  BIP122_SIGN_MESSAGE = "signMessage",
+  BIP122_SIGN_PSBT = "signPsbt",
```

Comment:
Variables are named with namespace to avoid misuse and conflicts with other chains. Also, In given connection request, methods are also separated by namespace e.g.
```
eip155: { methods: ['personal_sign']},
bip122: { methods: ['signMessage']}
```